app/assets/stylesheets/activities.css.scss

```diff
@@ -51,7 +51,7 @@ article {
 
 #hunting-card{
     height: 176px;
-    background: url('hunting.jpg') center / cover;
+    background: image-url('hunting.jpg') center / cover;
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Prefer double-quoted strings